Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabStrip.css

```diff
@@ -1,192 +1,70 @@
 /* Style applied to the TabStrip. */
 
-/* Applied to the tab strip itself.*/
-*.tabStrip
-{
-}
-
-*.tabStripTabsPane
-{
-}
-
-*.tabStripTabsPane.empty
-{
-}
-
-*.tabStripTabsPane ul
+.tabStripTabsPane ul
 {
+  padding: var(--remotion-themed-spacing);
   margin: 0;
-  padding: 0px;
   list-style: none;
-  width: auto;
-}
-
-*.tabStripTabsPane ul li
-{
-  display: inline;
+  display: flex;
+  flex-flow: wrap;
+  gap: var(--remotion-themed-spacing);
+  box-sizing: border-box;
 }
 
-*.tabStripTab a
+.tabStripTab a
 {
-  border: solid 1px #AAAAAA;
-  background-image: url(../Image/sprite.svg#ButtonBackground);
-  background-size: 1px 20px;
-  background-color: #E6E5E7;
-  color: Black;
-  text-decoration: none;
   cursor: default;
 }
 
-*.tabStripTab a[href]
+.tabStripTab a[href]
 {
-  color: Black;
-  cursor: hand;
+  color: var(--color-standard-text);
   cursor: pointer;
   outline: none;
+  text-decoration: none;
 }
 
-*.tabStripTab a:focus,
-*.tabStripTab a:hover
-{
-  color: Black;
-  border: solid 1px #7CB2E6;
-  background-image: url(../Image/sprite.svg#ButtonBackgroundHover);
-  background-size: 1px 20px;
-  background-color: #E5F0FC;
-}
-
-*.tabStripTab a:focus,
-*.tabStripTabSelected a:focus
+.tabStripTab a:focus,
+.tabStripTabSelected a:focus
 {
   outline: none;
-}
-
-*.tabStripTab a:focus span.anchorBody > img:only-child,
-*.tabStripTabSelected a:focus span.anchorBody > img:only-child
-{
-  outline: dotted black 1px;
-  outline: auto 5px -webkit-focus-ring-color;
-}
-
-*.tabStripTab a:focus span.anchorBody > span,
-*.tabStripTabSelected a:focus span.anchorBody > span
-{
   text-decoration: underline;
 }
 
-*.tabStripTab a:active
+.tabStripTabWrapper .tabStripTabSelected,
+.tabStripTabWrapper .tabStripTab
```

Comment:
We should be consistent, either add always or never. @iam-mholle  just suggested being a bit more qualified. please check with him.